apps/aesophia_http/src/aesophia_cors_middleware.erl

```diff
@@ -0,0 +1,75 @@
+%%%=============================================================================
+%%% @copyright (C) 2018, Aeternity Anstalt
```

Comment:
The year is wrong :-)

Reply:
Well, I guess the original source has been copy-written back in 2018 :)